src/System.Runtime.Extensions/src/ApiCompatBaseline.netcoreappaot.txt

```diff
@@ -0,0 +1,10 @@
+Compat issues with assembly System.Runtime.Extensions:
```

Comment:
Do we have issue(s) logged to add these to the other platforms?

Reply:
For most of the new methods, it's just a matter of CoreRT taking the changes and then getting an updated ProjectN build with the new bits.

For a couple of them (those marked as runtime calls), it requires updating the ProjectN compiler to support them (I'll log a bug on CoreRT to track).

Reply:
https://github.com/dotnet/corert/issues/6577